samples/javascript/clientInitialization/tokenCredentialAuthenticationProvider/README.md

```diff
@@ -6,9 +6,7 @@
 
     - [Documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-configure-app-access-web-apis#add-credentials-to-your-web-application).
 
-2. Rename [secrets.sample.ts](./typescript/secrets.sample.ts) to secrets.ts or rename [secrets.sample.js](./javascript/secrets.sample.js) to secrets.js
-
-3. Update your clientId, tenantId, scopes, clientSecret in the **secrets.ts/secrets.js** file.
+2. Update your clientId, tenantId, scopes, clientSecret in the code.
```

Comment:
nit, I suggest we change
`Update your clientId, tenantId, scopes, clientSecret in the code.`

to 

`Update your clientId, tenantId, scopes, clientSecret in index.js.`

Do we need to set the port as well?

Reply:
Done